TESTING.asciidoc

```diff
@@ -657,7 +619,7 @@ care.
 == Test coverage analysis
```

Comment:
I'm wonder if we can remove this whole section. Test coverage analysis is not part of our development process in any formal way. This assumes that contributors want some way of analyzing test coverage themselves? I'm not sure just saying "you're on your own" is even worth mentioning.

Reply:
Let's leave that for a separate issue, since this PR is about packaging test docs.